ppr-ui/src/components/parties/party/PartyAutocomplete.vue

```diff
@@ -183,7 +183,7 @@ export default defineComponent({
         addSecuredParty(newParty)
       }
 
-      context.emit('selectItem')
+      context.emit('selectItem', newParty)
```

Comment:
Emit autocompleted Party all the way up to the ContactInformation component, and then set the store.